### PR TITLE
Support 146 0.6.0

### PR DESCRIPTION
**Pull request by edwinvandenbelt**

### CODE REVIEW

File: TOMP-API.yaml [hunk 1]
```diff
@@ -996,29 +1001,21 @@ paths:
...
 
-  /support/{id}/request/{support-type}:
+  /support/:
     parameters:
       - $ref: '#/components/parameters/acceptLanguage'
       - $ref: '#/components/parameters/api'
       - $ref: '#/components/parameters/apiVersion'
     put:
-      description: identifies a request for support from user via MP
+      description: creates a request for support from end user via MP
```
Comment (ARBaart) [inline TOMP-API.yaml:1010]: Creating something is usually a post

File: TOMP-API.yaml [hunk 2]
```diff
@@ -908,27 +913,27 @@ paths:
         - payment
         - MP
         - TO
-      description: Returns all the journal entries that should be payed per leg
+      description: Returns all the journal entries that should be paid per leg
       parameters:
         - name: from
-          in: path
+          in: query
```
Comment (ARBaart) [inline TOMP-API.yaml:919]: I see this hotfix code is not already in develop-minor, shall I merge that first?

File: TOMP-API.yaml [hunk 3]
```diff
@@ -996,29 +1001,21 @@ paths:
         '401':
           $ref: '#/components/responses/401Unauthorized'
 
-  /support/{id}/request/{support-type}:
+  /support/:
```
Comment (ARBaart) [inline TOMP-API.yaml:1004]: What is the motivation of putting this in the develop-minor branch, when it seems like a breaking change? Is that how you want to split 0.6 vs 1.0?

File: TOMP-API.yaml (hunk 2 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:919]: please do so

File: TOMP-API.yaml (hunk 3 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:1004]: Nope. I try to do it in 2 steps.. 0.6 in next meeting, 1.0 in the meeting of june.

File: TOMP-API.yaml (hunk 1 above)
Comment (edwinvandenbelt) [reply TOMP-API.yaml:1010]: You're right. I'll change it.

**Pull request merged**